schema.json

```diff
@@ -1039,6 +1090,9 @@
           {
             "$ref": "#/$defs/SeqBlock"
           },
+          {
+            "$ref": "#/$defs/ParBlock"
```

Comment:
```suggestion
            "$ref": "#/$defs/QParBlock"
```

Reply:
BTW `schema.json` is autogenerated from the model using `make updateschema`. ⁠ :)

Reply:
Oh. lol oops :D